source/css/main.css

```diff
@@ -29,3 +29,8 @@ main {
   color: #fff;
   z-index: 10;
 }
+
+settings-menu {
```

Comment:
i think this can be achieved using `:host` pseudo-tag inside the web component declaration